src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -150,6 +150,18 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::PopStyleColor();
     }
 
+    // GPU
+    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+    if (ImGui::BeginTabItem("GPU")) {
+        ImGui::PopStyleColor();
+        ImGui::Checkbox("Hardware Flip", &host.cfg.hardware_flip);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Check the box to enable flipping from GPU side.\nMay increase performance on commercial games.\nIt is recommended to disable this option for homebrew.");
```

Comment:
Check the box to enable **texture** flipping from GPU side.
It is recommended to disable this option for homebrew **games**.

if this change is made I will be confident in merging

Reply:
no hombrew game, but just all hombrew ^^ for other oki :)